tools/monitoring/alert/alert.go

```diff
@@ -58,8 +58,8 @@ func (c *Client) RunAlerting() {
 }
 
 func (c *Client) handleReportMessage(rmsg *prowapi.ReportMessage) {
-	if rmsg.Status == prowapi.SuccessState || rmsg.Status == prowapi.FailureState || rmsg.Status == prowapi.AbortedState {
-		log.Printf("Received Pubsub message %v\n", rmsg)
+	if rmsg.Status == prowapi.FailureState || rmsg.Status == prowapi.AbortedState {
```

Comment:
Wouldn't it be safer to just use

```suggestion
	if rmsg.Status != prowapi.SuccessState {
```

?

Reply:
Not sure if the status can be other states, like `Pending` or `Triggered`. If yes, probably we should not make this change. Maybe @yt3liu can confirm?

Reply:
The status can definitely be `Pending` and `Triggered`. For the purpose of monitoring, it'd only care about final states.

Reply:
@chaodaiG an example would be resource not ready/found for long and the job times out

Reply:
@adrcunha Joyce is right. We should only check final state

Reply:
Thanks for clarifying.

/hold cancel